x/name/wasm/encode.go

```diff
@@ -22,6 +22,8 @@ type NameMsgParams struct {
 	Bind *BindNameParams `json:"bind_name,omitempty"`
 	// Encode a MsgUnBindName
 	Delete *DeleteNameParams `json:"delete_name,omitempty"`
+	// Encode a MsgCreateRootName
+	CreateRoot *CreateRootNameParams `json:create_root_name,omitempty`
```

Comment:
Missing the `"` around the stuff.
```suggestion
	CreateRoot *CreateRootNameParams `json:"create_root_name,omitempty"`
```